### PR TITLE
we are migrating to askama

### PR DESCRIPTION
**Pull request by tshepang**
see https://github.com/rust-lang/rust/issues/108868

### CODE REVIEW

File: src/librustdoc/html/templates/STYLE.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Askama templates support quite sophisticated control flow. To keep our templates
 simple and understandable, we use only a subset: `if` and `for`. In particular
 we avoid [assignments in the template logic][assignments] and [Askama
 macros][macros]. This also may make things easier if we switch to a different
-Jinja-style template system, like Askama, in the future.
+Jinja-style template system in future.
```
Comment (bugadani) [inline src/librustdoc/html/templates/STYLE.md:35]:
```suggestion
Jinja-style template system in the future.
```

Comment (tshepang) [reply src/librustdoc/html/templates/STYLE.md:35]: I find the word redundant

Comment (est31) [reply src/librustdoc/html/templates/STYLE.md:35]: Apparently [this is a british vs american english difference](https://www.oxfordlearnersdictionaries.com/us/definition/english/future_1?q=future), where BE uses "in future", and AE uses "in the future".

Comment (bugadani) [reply src/librustdoc/html/templates/STYLE.md:35]: @est31 note that "in future" is used as "from now on" in the idioms section, which has a different meaning than the sentence here, which refers to "at a later time" instead.

**Comment by tshepang** [comment]
'tis ready

**Review by est31 — APPROVED** [review]

**Pull request merged**